Rubberduck.Core/AutoComplete/AutoCompleteBase.cs

```diff
@@ -16,21 +16,24 @@ protected AutoCompleteBase(string inputToken, string outputToken)
 
         public virtual bool Execute(AutoCompleteEventArgs e)
         {
-            var selection = e.CodePane.Selection;
-            if (selection.StartColumn < 2) { return false; }
-
-            if (!e.IsCommitted && e.OldCode.Substring(selection.StartColumn - 2, 1) == InputToken)
+            using (var pane = e.CodePane)
```

Comment:
This using statement is dangerous. The code pane should be disposed by the creator of the event args and never by the receiver of the event args. The caller, `AutoCompleteService` still uses the code pane after the call to `Execute`.